src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2486,16 +2486,18 @@ export class StudyViewPageStore {
             };
         }
 
-        if (!_.isEmpty(this.mutationProfiles.result)) {
-            _chartMetaSet[UniqueKey.FUSION_GENES_TABLE] = {
-                uniqueKey: UniqueKey.FUSION_GENES_TABLE,
-                dataType: getChartMetaDataType(UniqueKey.FUSION_GENES_TABLE),
-                patientAttribute:false,
-                displayName: 'Fusion Genes',
-                priority: getDefaultPriorityByUniqueKey(UniqueKey.FUSION_GENES_TABLE),
-                renderWhenDataChange: true,
-                description: ''
-            };
+        if (!_.isEmpty(this.molecularProfileSampleCounts.result)) {
+            if (this.molecularProfileSampleCounts.result!.numberOfFusionProfiledSamples > 0) {
```

Comment:
not important but might as well just add this to wrapping if statement

Reply:
i.e. if (A && B) {  }

note that B will not evaluate if A is false

Reply:
good point. change made.